### DISCUSSION TITLE
since new release i cannot set my smtp credentials

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#1935] [Help]
Hello,

Once I filled my smt credentials, I click on save and a few moment later I have a upstream timeout error.

thanks a lot

**Comment by shogunpurple** [comment]
Hi @oswaldB - it seems like something may be timing out. 

Can you paste me the results of a `docker ps` in your terminal?

**Reply by oswaldB** [reply]
![image](https://user-images.githubusercontent.com/3023526/124610973-e8491200-de68-11eb-96e5-ce350fc4a414.png)

**Reply by shogunpurple** [reply]
This all looks good. It's possible something is wrong with the worker. The logs would be helpful:

`docker logs a41dc311d1c9`

**Comment by oswaldB** [comment]
ERROR    [1625658809787] (29 on 4b9fc8231224):
    err: {
      "type": "ForbiddenError",
      "message": "Unauthorized - no public worker access",
      "stack":
          ForbiddenError: Unauthorized - no public worker access
              at Object.throw (/app/node_modules/koa/lib/context.js:97:11)
              at /app/src/api/index.js:65:16
              at dispatch (/app/node_modules/koa-compose/index.js:42:32)
              at ***@***.***/router/lib/router.js:368:16
              at dispatch (/app/node_modules/koa-compose/index.js:42:32)
              at
***@***.***/auth/src/middleware/authenticated.js:73:14
              at dispatch (/app/node_modules/koa-compose/index.js:42:32)
              at ***@***.***/router/lib/router.js:368:16
              at dispatch (/app/node_modules/koa-compose/index.js:42:32)
              at /app/node_modules/koa-compress/lib/index.js:38:11
      "status": 403,
      "statusCode": 403,
      "expose": true
    }
    req: {
      "id": 122,
      "method": "GET",
      "url": "/api/admin/users/self",
      "headers": {
        "host": "bruxelles.av42.com",
        "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0)
Gecko/20100101 Firefox/78.0",
        "accept": "*/*",
        "accept-encoding": "gzip, deflate, br",
        "accept-language": "fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3",
        "content-type": "application/json",
        "referer": "https://bruxelles.av42.com/builder",
        "te": "trailers",
        "x-budibase-app-id": "",
        "x-forwarded-for": "62.210.39.136",
        "x-forwarded-proto": "https",
        "x-request-id": "5e3056ec-7f51-4d7b-99a5-a808fb1c55ef",
        "x-envoy-expected-rq-timeout-ms": "15000",
        "content-length": "0"
      },
      "remoteAddress": "::ffff:172.18.0.5",
      "remotePort": 32896
    }

On Tue, 6 Jul 2021 at 17:03, Martin McKeaveney ***@***.***>
wrote:

> This all looks good. It's possible something is wrong with the worker. The
> logs would be helpful:
>
> docker logs a41dc311d1c9
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/1935#discussioncomment-970888>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXCFJTRWB2UHA3Y4GJWJDLTWMSM5ANCNFSM474QCNJQ>
> .
>


-- 
Oswald BERNARD
00 44 7861004200
